applications/pfem_2_application/custom_utilities/particle_utilities.h

```diff
@@ -114,14 +115,11 @@ namespace Kratos
 	    
             double courant= norm_u * max_dt / h;
 	    
-	    
-            double& counter = im->GetValue(POISSON_RATIO);
+	    double& counter = im->GetValue(POISSON_RATIO);
```

Comment:
why are u using the poisson ratio here? also, why do you update the elemental value?